gcc/testsuite/rust/execute/torture/match-restpattern-tuple.rs

```diff
@@ -0,0 +1,13 @@
+fn main() -> i32 {
```

Comment:
can you copy some of the other test cases to use a dg-output comment at the top and use a call to puts or printf to show that its outputing what you expect?

Oh neat i see you are using the ret variable to do this to make sure we return 0 i think it would be good to also add in a puts call here too